guvnor-ala/guvnor-ala-services-rest/src/test/java/org/guvnor/ala/services/rest/tests/RestRuntimeProvisioningImplTest.java

```diff
@@ -132,6 +136,11 @@ public static Archive createDeployment() throws Exception {
         return deployment;
     }
 
+    @BeforeClass
+    public static void beforeClass() {
+        Assume.assumeFalse(System.getProperty("os.name").toLowerCase(Locale.ENGLISH).contains("windows"));
```

Comment:
Any chance we could share the conditional check with the `Requirement` below? Perhaps some utility method, such as `isWindows()`?